src/main/java/frc/robot/commands/FindAndGotoNote.java

```diff
@@ -133,20 +157,27 @@ else if(target.getYaw() > angleTolerance) {
             }
 
             else {
-//                System.out.println("Autonomous Not Getting Note");
+                DataLogManager.log("Autonomous Not Getting Note");
             }
+
+//            DataLogManager.log("Needed Speed: " + neededSpeed + ", Needed Rotation: " + neededRotation);
         }
         else {
-//            System.out.println("No Note in view...");
+            System.out.println("No Note in view...");
             neededSpeed = 0.0;
             neededRotation = 0.0;
             // blink the blinkin
             RobotState.getInstance().setNoteIsAvailable(false);
+
+            if (DriverStation.isTeleop()) {
+                RobotState.getInstance().setNoteIsAvailable(true);
```

Comment:
why are we setting note is available to true in the case where there was no target? that makes no sense to me...

Reply:
That was to try to get it to still spin the intake, so that in the case that the auto-align failed, we would still be able to intake, but it's overwritten by the end method anyway, so this doesn't actually do anything.